src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('span.population');
+const averagePopulation = document.querySelector('span.average-population');
+const totalPopulation = document.querySelector('span.total-population');
+const numbers = Array.from(populations).map((span) => {
+  const row = span.textContent;
+
+  return +row.split(',').join('');
+});
+
+const total = numbers.reduce((sum, num) => sum + num, 0);
+const average = total / numbers.length;
```

Comment:
Ensure that `numbers.length` is not zero to avoid division by zero errors. Although this is unlikely with valid data, it's a good practice to handle such cases.